app/assets/javascripts/Components/Result/result.jsx

```diff
@@ -505,6 +505,7 @@ class Result extends React.Component {
           marking_state={this.state.marking_state}
           num_assigned={this.state.num_assigned}
```

Comment:
- `num_assigned` should no longer be necessary, remove it here and everywhere else